documentation.html

```diff
@@ -60,6 +60,81 @@
                 <!-- for example -->
                 <!-- <div class="compcontainer"></div> -->
 
+                <div id="cards">
+                    <h1 class="md-title">Cards</h1>
+                    <p class="rg-p mg-t-10">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur, et laborum voluptate modi,
+                        nulla ex, commodi rem dolor quas alias est quae sequi corporis eum mollitia aperiam iure cumque
+                        nam?
+                    </p>
+
+                    <div class="mg-t-30">
+                        <h2 class="rg-booky-title">Horizontal Cards with Image and Text</h2>
+                        <div class="center-block card-ex">
+                            <div class="flex-col mg-t-30">
+                                <div class="card hor-card">
+                                    <div class="img-container">
+                                        <img src="./Assets/Images/book.jpg" alt="">
+                                    </div>
+                                    <div class="text-card">
+                                        <p class="rg-booky-title">The Chronicles of Narnia </p>
+                                        <p class="xsm-p grey-txt">Written by C.S.Lewis</p>
+                                        <p class="sm-p mg-t-10">Lorem ipsum dolor sit amet consectetur, adipisicing
+                                            elit. Fugit
+                                            consequatur officiis doloribus. .</p>
+                                        <div class="review-row mg-t-20">
+                                            <div class="rating-row">
+                                                <i class="fas fa-star"></i>
+                                                <p class="sm-title">4.3</p>
+                                            </div>
+                                            <i class="fas fa-circle"></i>
+                                            <p class="review-num xsm-p grey-txt">(43)</p>
+                                        </div>
+                                        <div class="money mg-t-10">
+                                            <i class="fas fa-rupee-sign"></i>
+                                            <p class="rg-title">200.83</p>
```

Comment:
The font-size of the rupee icon can be made smaller, if this is the original size intention

Reply:
im going to use Rs. instead